src/components/guide/index.js

```diff
@@ -135,8 +150,8 @@ class Guide extends Component {
     }
   }
   render() {
-    if(this.state.errors.length > 0) { return <div />; }
-    if(this.state.competencies.length === 0) { this.setGuide(); return <div />; }
+    if(this.state.errors.length > 0) { this.processError(); return <div />; }
```

Comment:
You'd want to do this wherever errors gets set, otherwise when any prop or state change occurs, the same error will be sent again

Reply:
Good thought, I updated it. Let me know if I didn't handle this update the way you would expect